inquire/src/prompts/path_select/mod.rs

```diff
@@ -0,0 +1,365 @@
+mod action;
+use action::*;
+mod path_entry;
+pub use path_entry::PathEntry;
+mod prompt;
+use prompt::*;
+mod config;
+use config::*;
+mod modes;
+pub use modes::*;
+#[cfg(feature = "crossterm")]
+#[cfg(test)]
+mod test;
+
+use crate::{
+    config::get_configuration,
+    error::InquireResult,
+    formatter::MultiOptionFormatter,
+    list_option::ListOption,
+    prompts::prompt::Prompt,
+    terminal::get_default_terminal,
+    type_aliases::Filter,
+    ui::{Backend, MultiSelectBackend, RenderConfig},
+    InquireError,
+};
+use std::path::Path;
+
+/// Prompt for choosing one or multiple files.  
+///
+/// The user can
```

Comment:
Let me know what you think. One thing I did not document was the divider. It is currently unused because I couldn't figure out how to make it work with the prompt — basically, I thought it would be less confusing with a visual divider between the current directory's entries and items selected from previous directories if any. Only I don't know how to insert a divider into the list or if that's possible, so maybe you can suggest something?